File: backend/WorldBeauty/Servico.java
package WorldBeauty;

public class Servico {
    private String nome;
    private float valor;

    Servico(String nome, float valor) {
        this.nome = nome;
        this.valor = valor;
    }

    public String GetNome() {
        return this.nome;
    }
    public float GetValor() {
        return this.valor;
    }

    public void SetValor(float valor) {
        this.valor = valor;
    }
}

File: world-beauty/src/WB-ATV1/src/Models/Servico.ts
export default class Servico {
    private nome: string;
    private valor: number;

    constructor(nome: string, valor: number) {
        this.nome = nome;
        this.valor = valor;
    }

    public get GetNome(): string {
        return this.nome;
    }
    public get GetValor(): number {
        return this.valor;
    }

    public set SetValor(valor: number) {
        this.valor = valor;
    }
}